model/shared/src/main/scala/explore/model/GlobalPreferences.scala

```diff
@@ -32,6 +32,9 @@ object ElevationPlotScheduling extends NewType[Boolean]:
 
 type ElevationPlotScheduling = ElevationPlotScheduling.Type
 
+given Decoder[GraphType] = Decoder.decodeString.map:
+  case "S2N_CHART" => GraphType.S2NGraph
```

Comment:
What about the ohter cases?

Reply:
Good catch. This was meant to be temporary. The idea is to use a migration to rename in the DB and use the standard codecs. I'll add a commit with that.